end_to_end/1-data-prep-e2e.ipynb

```diff
@@ -0,0 +1,1401 @@
+{
```

Comment:
<p>Remove the # symbols.</p><p> </p><p>Keep the ToC for Notebook 1 here.</p><p>This complete ToC should be in the 0th notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1974/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>agreed</p>